### ISSUE TITLE
Training accuracy on Kinetics

### ISSUE THREAD

**Issue by cryptedp** [kenshohara/3D-ResNets-PyTorch#29]
Hi
Since I cannot find, what training accuracy you achieved on the Kinetics training set, I am not sure if the accuracy I obtained is high enough?
I obtain around 30% training accuracy, however the loss is similar to  that reported in the paper (around 3), so I am not sure if the accuracy is also similiar ?

**Comment by kenshohara** [comment]
When I trained ResNet-18,
training cross-entropy loss and accuracy are 2.87 and 37.3% at epoch 150, respectively.
When I continued the training until epoch 250,
the loss and accuracy are 2.68 and 40.9%, respectively.

**Comment by KaiyuYue** [comment]
Does that means when training loss and accuracy are about 2.68 and 40.9% respectively, the network is almost getting the state of convergence, and it can give the test accuracy around 70%?

**Comment by kenshohara** [comment]
Validation accuracy is 44% at epoch 250.
These accuracies are per clip accuracies.
Its top-1 and top-5 video accuracy is 54% and 78% on the validation set, respectively.

**Comment by WUSHUANGPPP** [comment]
@kenshohara I trained resnet18 of initial lr 0.05 for 300 epochs. But I got the score of 47% video-accuracy evaluated by the inference.py. I don't know if it is the reason of different trianing hyperparameters.
